modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -1410,6 +1410,34 @@ TEST_P(Test_ONNX_layers, LSTM_layout_batch)
     testONNXModels("lstm_layout_1", npy, 0.005, 0.005, false, false, 3);
 }
 
+TEST_P(Test_ONNX_layers, Einsum_2D)
```

Comment:
Can we add the onnx conformance tests as well? There are 5 tests from the onnx repo:

```
$ONNX/onnx/backend/test/data/node/test_einsum_sum
$ONNX/onnx/backend/test/data/node/test_einsum_inner_prod
$ONNX/onnx/backend/test/data/node/test_einsum_batch_diagonal
$ONNX/onnx/backend/test/data/node/test_einsum_transpose
$ONNX/onnx/backend/test/data/node/test_einsum_batch_matmul
```